addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -531,7 +531,7 @@ export const editorCommands = {
             node => closestElement(node).isContentEditable
         );
         let coloredTds = [];
-        if (selectedTds.length) {
+        if (selectedTds.length && ["backgroundColor", "background-color"].includes(mode)) {
```

Comment:
It seems very weird to me that we could end up with both those values here. I think only one of them should be supported and therefore the other is a bug that should not happen. From the diff it is hard to detect which one is the correct one though, but I'd be quite surprised if `colorElement` actually supported both of those ?

Reply:
> From the diff it is hard to detect which one is the correct one though

The docstring says `@param {string} mode 'color' or 'backgroundColor'`.

> I'd be quite surprised if `colorElement` actually supported both of those

[Indeed it doesn't](https://github.com/odoo/odoo/pull/130646/files#diff-9106b7a2efaf668f75c6e3c76a67383d85b1b90b768454863663a7aa564862c3L89).

Reply:
No but I found a case where it happen, let me retest it.

Reply:
I quickly grepped our calls to `applyColor` and found the string is always hardcoded. What there is though is a mistake in a test ^^'
![image](https://github.com/odoo/odoo/assets/24205914/293c2295-6b83-4c5c-a7e4-3a6f061a9920)
The test kind of passes by accident :-P

Here: https://github.com/odoo-dev/odoo/blob/37fdc8e35bf735c4a70d874e730b47ea4cdfe8ca/addons/web_editor/static/src/js/editor/odoo-editor/test/spec/color.test.js#L58

Reply:
Cool then let's go